app/models/playbook.rb

```diff
@@ -21,12 +21,10 @@ class Playbook < ApplicationRecord
   has_many :ratings, dependent: :destroy
   validates :name, presence: true
 
+  attribute(:config, :playbook_config)
```

Comment:
I would recommend setting the default to be explicitly nil
```suggestion
  attribute :config, :playbook_config, default: nil
```